symbulate/random_variables.py

```diff
@@ -73,6 +73,32 @@ def sim(self, n):
 
         return RVResults(self.draw() for _ in range(n))
 
+    def __call__(self, input):
+        print("Warning: Calling an RV as a function simply applies the function that defines\n"
+            "the RV to the input, regardless of whether the input is a valid outcome in \n"
+            "the underlying probability space.")
+        dummy_draw= self.probSpace.draw()
+        if isinstance(input, tuple):
+            if not isinstance(dummy_draw, (tuple, list)):
+                raise Exception("The underlying probability space returns a single value. "
+                "A(n) " + type(input).__name__ + " was given.")
```

Comment:
If I remember correctly, str(type(input)) would return something like <class 'int'>. In order to just get int, you need to call the __name__ attribute.